ci-operator/config/openshift-priv/cluster-etcd-operator/openshift-priv-cluster-etcd-operator-release-4.12.yaml

```diff
@@ -27,6 +27,10 @@ base_images:
     name: 4.12-priv
     namespace: ocp-private
     tag: base
+  cluster-api-actuator-pkg-test:
+    name: cluster-api-actuator-pkg-test
+    namespace: ci
+    tag: "4.12"
```

Comment:
Do we really want to update this file ?

Reply:
There is a job in this file used chain openshift-e2e-test-qe, 
```
- as: e2e-gcp-qe-no-capabilities
  optional: true
  steps:
    cluster_profile: gcp-qe
    env:
      BASELINE_CAPABILITY_SET: None
    test:
    - chain: openshift-e2e-test-qe
    workflow: cucushift-installer-rehearse-gcp-upi
```
instead of adding image, I can update the test steps as below, do you think which way is better?
```
  - chain: cucushift-installer-check-cluster-health
  - ref: idp-htpasswd
  - ref: fips-check-fips-or-die
  - ref: fips-check-node-scan
  - ref: cucushift-pre
  - ref: openshift-extended-test
  - ref: cucushift-e2e
  - ref: openshift-extended-web-tests
  - ref: openshift-extended-test-supplementary
  - ref: openshift-e2e-test-clusterinfra-qe
  - ref: openshift-e2e-test-qe-report
```

Reply:
AFAIK, openshift-priv is used by ART team, and there are some auto mechanism there. So generally we don't  made changes manually there, unless someone reach to us and ask us to fix something.

Reply:
Removed, please help to take a look again.